src/plugins/wb-disable/disable.js

```diff
@@ -53,11 +67,19 @@ var componentName = "wb-disable",
 						localStorage.setItem( "wbdisable", "true" );
 					} catch ( e ) {}
 
-					// Append the Standard version link
-					li.innerHTML = "<a class='wb-sl' href='" + nQuery + "wbdisable=false'>" + i18n( "wb-enable" ) + "</a>";
+					// Temporary fix for i18n until text are translated
+					if ( !noticeHeader && wb.lang === "fr" ) {
+						noticeHeader = i18nTemporary.fr[ "disable-notice-h" ];
+						noticeBody = i18nTemporary.fr[ "disable-notice" ];
+					} else if ( !noticeHeader ) {
+						noticeHeader = i18nTemporary.en[ "disable-notice-h" ];
+						noticeBody = i18nTemporary.en[ "disable-notice" ];
+						noticehtml = noticehtml + " lang='en'";
+					}
```

Comment:
The current i18n framework defaults to the english when no translation is provided and doesn't add the `lang=en` so that is a wider issue.